modules/pam_access/pam_access.c

```diff
@@ -737,7 +737,9 @@ network_netmask_match (pam_handle_t *pamh,
 		{ /* invalid netmask value */
 		  return NO;
 		}
-	    if ((netmask < 0) || (netmask >= 128))
+	    if ( (netmask < 0) ||
```

Comment:
Just please fix these formatting nits: Do not put spaces after the initial parenthesis after if and before the last parenthesis. Put spaces around `==` and move the `||` at the beginning of the line for better readability.

Reply:
Hi, thanks for the comments, I've fixed the formatting.